src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -43,45 +48,52 @@ const DevSecurity = ({ children }: ParentComponentProps) => {
   useEffect(() => {
     if (window.localStorage[storageKey]) {
       const state = JSON.parse(window.localStorage[storageKey]);
-      setAuthState(as => {
+      setAuthState(prevAuthState => {
         return {
-          ...as,
-          name: `User ${state.eua}`,
-          isAuthenticated: true,
-          euaId: state.eua,
-          groups: state.jobCodes
+          ...prevAuthState,
+          isAuthenticated: true
         };
       });
+      setEuaId(state.eua);
+      setGroups(state.jobCodes);
     }
   }, []);
 
-  const handleSubmit: ReactEventHandler = event => {
```

Comment:
In this line, `ReactEventHandler` is the return value of `handleSubmit`. I'm not sure if that's the intent, but how I'm reading it is that the `event` argument is supposed to be `React.FormEvent`

Reply:
I think that ReactEventHandler was the type of the function vs. the type of the return value? But in any case I think the newer code is an improvement since it's generally better to declare the types of arguments and return values than functions themselves.

Reply:
Oh you're right! The return type is after the arguments.